src/main/com/aca/homework/week7/button/click/Button.java

```diff
@@ -0,0 +1,13 @@
+package com.aca.homework.week7.button.click;
+
+public class Button {
+    private final String text;
+
+    public Button(String text) {
+        this.text = text;
+    }
+
+    public void click(Action action) {
```

Comment:
the action should not be passed as an argument to the click method. instead, pass it through the constructor